src/app/services/coin.service.ts

```diff
@@ -3,35 +3,37 @@ import { BehaviorSubject } from 'rxjs/BehaviorSubject';
 import { BaseCoin } from '../coins/basecoin';
 import { SkycoinCoin } from '../coins/skycoin.coin';
 import { TestCoin } from '../coins/test.coin';
+import { coinsId } from '../constants/coins-id.const';
 
 export class CoinService {
 
   currentCoin: BehaviorSubject<BaseCoin> = new BehaviorSubject<BaseCoin>(null);
+  coins: BaseCoin[] = [];
 
-  private coins: BaseCoin[] = [];
   private readonly storageKey = 'currentCoin';
+  private readonly defaultCoinId = coinsId.sky;
```

Comment:
The defailt coin is `defaultCoinId`, from `../constants/coins-id.const`.

Reply:
Fixed.